### ISSUE TITLE
Keyboard inputs not recognized by Blockly when NVDA is in browse mode

### ISSUE THREAD

**Issue by aboubakar-mtp** [RITAccess/blockly#64]
### Problem statement

With NVDA turned on, Blockly accepts keyboard inputs only when NVDA is in Focus mode. After you add a block from the toolbox to the workspace, NVDA automatically switches back to browse mode and all subsequent key inputs are not received by Blockly. For it to work again, you have to toggle NVDA back to focus mode.

A workaround this is to set NVDA to Focus mode (using NVDA commands) upon page load. This maintains NVDA in Focus mode and prevents it from automatically toggling to browse mode when you add a block to the workspace.

### Expected Behavior

Keyboard inputs should be accepted and processed with NVDA turned on in any mode

### Steps to Reproduce

1. Start by..
2. Next, do..

### Stack Traces

```
Replace with error stack trace.
```
